packages/suite/src/views/wallet/send/components/Outputs/components/Amount/components/Fiat.tsx

```diff
@@ -192,7 +192,10 @@ export const Fiat = ({ output, outputId }: FiatProps) => {
                 // Get (fresh) fiat rates for newly selected currency
                 const updateFiatRatesResult = await dispatch(
                     updateFiatRatesThunk({
-                        ticker: { symbol: account.symbol as NetworkSymbol },
+                        ticker: {
+                            symbol: account.symbol as NetworkSymbol,
+                            tokenAddress: (token?.contract as TokenAddress) ?? undefined,
```

Comment:
```suggestion
                            tokenAddress: token?.contract as TokenAddress
```
Does the same job right?

Reply:
d12a573208bc86ea095dac7106213470b4e2b0f2